demo-app/src/main/java/org/openmined/syft/demo/domain/MNISTDataRepository.kt

```diff
@@ -1,7 +1,7 @@
 package org.openmined.syft.demo.domain
 
 import org.openmined.syft.demo.datasource.LocalMNISTDataDataSource
-import java.util.ArrayList
+import java.util.*
```

Comment:
Keep the specific import.
I think this has to be set in the preferences.

Reply:
Yeah will do